editor/blocks/quote/style.scss

```diff
@@ -0,0 +1,24 @@
+#editor blockquote {
+	margin: 2em;
```

Comment:
Yep, I guess the other one had a margin in all the blocks. The margin felt a bit weird while testing.